build.gradle

```diff
@@ -18,7 +18,9 @@ ext {
     MIN_SDK_VERSION = 15
     SDK_VERSION = 28
     BUILD_TOOLS_VERSION = '28.0.3'
-    SUPPORT_LIBRARY_VERSION = '28.0.0'
+    ANDROIDX_CORE_VERSION = '1.0.0'
+    ANDROIDX_ANNOTATION_VERSION = '1.0.0'
+    ANDROIDX_RECYCLERVIEW_VERSION = '1.0.0'
```

Comment:
[Q] Should we specify each version instead of the single `ANDROIDX_VERSION`?

Reply:
https://developer.android.com/jetpack/androidx/androidx-rn

It seems that AndroidX is a set of libraries that have the only one version, so  `ANDROIDX_VERSION` seems enough.

Reply:
AndroidX starts semantic versioning per library.
https://android-developers.googleblog.com/2018/05/hello-world-androidx.html

Currently, all artifacts is the same version but it will be different in the future.

Reply:
I see! okay!